pydy/system.py

```diff
@@ -96,6 +96,8 @@ class System(object):
     """
     def __init__(self, eom_method, constants=None, specifieds=None,
                  ode_solver=None, initial_conditions=None, times=None):
+        raise FutureWarning('PyDy System is experimental and may change in the future.')
```

Comment:
See how Oliver and I just changed the other warnings in the lastest PRs. You never "raise" a warning. You'll need to create a `PyDyFutureWarning` in `utils.py`.